### PR TITLE
adding meta header

### PR DESCRIPTION
**Pull request by danield137**

### CODE REVIEW

File: data-explorer/kusto/api/golang/kusto-golang-client-library.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+title: Azure Data Explorer Go SDK - Azure Data Explorer | Microsoft Docs
+description: This article describes Azure Data Explorer Go SDK in Azure Data Explorer.
+services: data-explorer
+author: dadubovs
```
Comment (orspod) [inline data-explorer/kusto/api/golang/kusto-golang-client-library.md:4]:
```suggestion
author: danield137
```

File: data-explorer/kusto/api/golang/kusto-golang-client-library.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+title: Azure Data Explorer Go SDK - Azure Data Explorer | Microsoft Docs
```
Comment (orspod) [inline data-explorer/kusto/api/golang/kusto-golang-client-library.md:1]:
```suggestion
title: Azure Data Explorer Go SDK
```

File: data-explorer/kusto/api/golang/kusto-golang-client-library.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+description: This article describes Azure Data Explorer Go SDK in Azure Data Explorer.
```
Comment (orspod) [inline data-explorer/kusto/api/golang/kusto-golang-client-library.md:2]:
```suggestion
description: This article describes Azure Data Explorer Go SDK.
```

File: data-explorer/kusto/api/golang/kusto-golang-client-library.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+services: data-explorer
```
Comment (orspod) [inline data-explorer/kusto/api/golang/kusto-golang-client-library.md:3]:
```suggestion
```

File: data-explorer/kusto/api/golang/kusto-golang-client-library.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
...
+author: dadubovs
+ms.author: dadubovs
+ms.reviewer: orspodek
+ms.service: data-explorer
+ms.topic: reference
+ms.date: 16/04/2020
```
Comment (orspod) [inline data-explorer/kusto/api/golang/kusto-golang-client-library.md:9]:
```suggestion
ms.date: 04/16/2020
```

**Comment by PRMerger17** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

**Pull request closed without merging**